packages/flutter/lib/src/material/autocomplete.dart

```diff
@@ -193,6 +194,12 @@ class Autocomplete<T extends Object> extends StatelessWidget {
   /// default.
   final AutocompleteOptionsViewBuilder<T>? optionsViewBuilder;
 
+  /// Defines the maximum height the options can take
+  /// if optionsViewBuilder is null
+  ///
+  /// Defaults to 200.0
```

Comment:
Nit: period at the end of each paragraph. The last line could just be `Defaults to 200.`